src/libraries/System.Net.Primitives/tests/FunctionalTests/CookieTest.cs

```diff
@@ -226,7 +226,8 @@ public static void Secure_GetSet_Success()
         [Fact]
         public static void Timestamp_GetSet_Success()
         {
-            DateTime dt = new DateTime(DateTime.Now.Year, DateTime.Now.Month, DateTime.Now.Day, DateTime.Now.Hour, DateTime.Now.Minute, 0); //DateTime.Now changes as the test runs
+            //DateTime.UtcNow changes as the test runs
+            DateTime dt = new DateTime(DateTime.UtcNow.Year, DateTime.UtcNow.Month, DateTime.UtcNow.Day, DateTime.UtcNow.Hour, DateTime.UtcNow.Minute, 0);
```

Comment:
The code reads current UtcNow repeatedly, so there is a possible race if few properties are red at e.g. 10:59 and others at 11:00. IIRC arguments are guaranteed to be evaluated left to right, so (in theory) we can't end up with 11:59, but since we are touching the code we might as well fix it.

```suggestion
            DateTime dt = DateTime.UtcNow;
            dt = new DateTime(dt.Year, dt.Month, dt.Day, dt.Hour, dt.Minute, 0);
```